docs/node-mixin/dashboards/node.libsonnet

```diff
@@ -126,28 +129,59 @@ local gauge = promgrafonnet.gauge;
           ],
         };
 
-      // TODO: It would be nicer to have a gauge that gets a 0-1 range and displays it as a percentage 0%-100%.
-      // This needs to be added upstream in the promgrafonnet library and then changed here.
-      // TODO: Should this be partitioned by mountpoint?
-      local diskSpaceUsage = gauge.new(
-        'Disk Space Usage',
-        |||
-          100 -
-          (
-            sum(node_filesystem_avail_bytes{%(nodeExporterSelector)s, instance="$instance", %(fsSelector)s})
-          /
-            sum(node_filesystem_size_bytes{%(nodeExporterSelector)s, instance="$instance", %(fsSelector)s})
-          * 100
-          )
-        ||| % $._config,
-      ).withLowerBeingBetter();
+      // TODO: Somehow partition this by device while excluding read-only devices.
```

Comment:
there's a metric for read only devices: https://github.com/prometheus/node_exporter/blob/7ef6f2576d5bef2fabb9e033a92da4d5a835915b/collector/filesystem_common.go#L105-L107

it should be possible to use it to ignore others

Reply:
Yes, certainly.

I just wanted to only do "trivial" things in this PR and leave anything even slightly more elaborate to later PRs, for the sake of not overloading one PR with too many things.

Reply:
Works for me :) thanks!